src/mmw/apps/geoprocessing_api/urls.py

```diff
@@ -36,5 +36,7 @@
     re_path(r'jobs/' + uuid_regex, get_job, name='get_job'),
     re_path(r'modeling/worksheet/$', views.start_modeling_worksheet,
             name='start_modeling_worksheet'),
+    re_path(r'modeling/gwlf-e/prepare/$', views.start_modeling_gwlfe_prepare,
```

Comment:
We use the hyphenated form `gwlf-e` in the URL as that is standardized. See the repository https://github.com/WikiWatershed/gwlf-e and the pypi package https://pypi.org/project/gwlf-e/